index.html

```diff
@@ -1,9 +1,11 @@
+<!DOCTYPE html>
 <html>
   <head>
     <title>Kanji Sheets</title>
-    <link href="/css/main.css" rel="stylesheet" type="text/css" />
+    <meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
+    <link href="css/main.css" rel="stylesheet" type="text/css" />
```

Comment:
Uhh, as far as I know, css/main.css will reference http://jensuchu.github.io/kanji/css/main.css since the index.html page is in http://jensuchu.github.io/kanji/.